app/DataTransferObjects/LicenseManagement/LicenseLimitsData.php

```diff
@@ -46,6 +46,7 @@ class LicenseLimitsData extends Data
 {
     public function __construct(
         public int $conversationalAiSeats,
+        public int $conversationalAiAssistants,
```

Comment:
Is it possible to set this equal to `0` here?

I believe that will set it as `0` as a default, which would prevent the error from happening if it was loaded before the migration ran.

Reply:
Sure, but params with defaults must come after params without, so I have now added defaults to all params to preserve the logical order of params